deploy/helm/distributed-compute-operator/templates/_helpers.tpl

```diff
@@ -1,90 +1,68 @@
-{{- else }}
-{{- $name := default .Chart.Name .Values.nameOverride }}
-{{- if contains $name .Release.Name }}
-{{- .Release.Name | trunc 63 | trimSuffix "-" }}
-{{- else }}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
-{{- end }}
-{{- end }}
-{{- end }}
-
-{{/*
-Create chart name and version as used by the chart label.
-*/}}
-{{- define "dco.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
-{{- end }}
-
-{{/*
-Common labels
-*/}}
-{{- define "dco.labels" -}}
-helm.sh/chart: {{ include "dco.chart" . }}
-{{ include "dco.selectorLabels" . }}
-{{- if .Chart.AppVersion }}
-app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
-{{- end }}
-app.kubernetes.io/managed-by: {{ .Release.Service }}
-{{- end }}
-
-{{/*
-Selector labels
-*/}}
-{{- define "dco.selectorLabels" -}}
-app.kubernetes.io/name: {{ include "dco.name" . }}
-app.kubernetes.io/instance: {{ .Release.Name }}
-{{- end }}
+{{- define "dco.image" -}}
+{{- $imageRoot := .Values.image -}}
+{{- $_ := set $imageRoot "tag" (.Values.image.tag | default .Chart.AppVersion) -}}
+{{- include "common.images.image" (dict "imageRoot" $imageRoot "global" $) -}}
+{{- end -}}
 
 {{/*
 Create the name of the service account to use
 */}}
 {{- define "dco.serviceAccountName" -}}
-{{- if .Values.serviceAccount.create }}
-{{- default (include "dco.fullname" .) .Values.serviceAccount.name }}
-{{- else }}
-{{- default "default" .Values.serviceAccount.name }}
-{{- end }}
-{{- end }}
+{{- if .Values.serviceAccount.create -}}
+{{- default (include "common.names.fullname" .) .Values.serviceAccount.name -}}
+{{- else -}}
+{{- default "default" .Values.serviceAccount.name -}}
+{{- end -}}
+{{- end -}}
 
 {{/*
 Webhook service name
 */}}
 {{- define "dco.webhook.service" -}}
-{{ include "dco.fullname" . }}-webhook-server
-{{- end }}
+{{ include "common.names.fullname" . }}-webhook-server
+{{- end -}}
 
 {{/*
 Webhook certificate CA name
 */}}
 {{- define "dco.webhook.issuer" -}}
-{{ include "dco.fullname" . }}-selfsigned-issuer
-{{- end }}
+{{ include "common.names.fullname" . }}-selfsigned-issuer
+{{- end -}}
 
 {{/*
 Webhook certificate name
 */}}
 {{- define "dco.webhook.certificate" -}}
-{{ include "dco.fullname" . }}-webhook
-{{- end }}
+{{ include "common.names.fullname" . }}-webhook
+{{- end -}}
 
 {{/*
 Webhook certificate secret name
 */}}
 {{- define "dco.webhook.secret" -}}
-{{ include "dco.fullname" . }}-webhook-cert
-{{- end }}
+{{ include "common.names.fullname" . }}-webhook-cert
+{{- end -}}
+
+{{/*
+Webhook certificate injection annotation
+*/}}
+{{- define "dco.webhook.annotation" -}}
+cert-manager.io/inject-ca-from: {{ .Release.Namespace }}/{{ include "dco.webhook.certificate" . }}
+{{- end -}}
+
+{{/*
+Returns a name suitable for all manager RBAC objects
+*/}}
+{{- define "dco.rbac.managerName" -}}
+dominodatalab:operator:{{ include "common.names.fullname" . }}:manager
```

Comment:
This might need to be a bit more "global" if we support multiple installs (since you can have the same release in two namespaces)

Reply:
see https://github.com/dominodatalab/distributed-compute-operator/pull/22